### DISCUSSION TITLE
4 star-only characters inquiry

### DISCUSSION THREAD

**Discussion by DiZeroX** [BeaverHouse/aecheck-front#45] [Questions] [answered]
### Category

Question

### Detailed description

Is it intended for 4 star characters without a NS such as Breeno to be excluded from the Character Checker?

**Comment by HU-Lee** [comment, marked as answer]
Yes, it's intended.

It was excluded since the first of the AE Check because it is quite natural to get all of the 3-4 stars character when you play AE long time, and the ability between the 5-star characters and the others are too huge so that very few people cares about that when playing the game.

If the character get any of the 5-star styles, including AS like Parisa recently, it will be added to the checklist. Also, free characters like Wryz will be added after 5-star is unlocked by the story update.

**Comment by DiZeroX** [comment]
Thanks for the response!
